### DISCUSSION TITLE
Unlimited results for a channel

### DISCUSSION THREAD

**Discussion by Ruygazunc** [FreeTubeApp/FreeTube#4521] [General]
Have to say I love this app. One thing I don't see though is a way to set an unlimited number of results for a Channel>Videos tab. There's no continuous scroll as on YouTube; you have to keep clicking "Fetch more results." Would be great if there was an option at least for continuous scroll. Thanks, I appreciate the work you've put into this project. Light years better than YouTube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE do you think we can maybe do something similar here like you done in https://github.com/FreeTubeApp/FreeTube/pull/3352

**Reply by PikachuEXE** [reply]
One toggle for all tabs? What should be the new option be called?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Toggle should be called infinite scroll (that's how i remember this feature being called)

It should be applied on all pages that have a load more button, when it reaches the bottom of the page it will auto load the content

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Will be implemented by https://github.com/FreeTubeApp/FreeTube/pull/4565

**Reply by Ruygazunc** [reply]
Right on. Love you guys.
